libtiledbvcf/src/CMakeLists.txt

```diff
@@ -135,20 +135,43 @@ endif()
 add_library(tiledbvcf SHARED
   $<TARGET_OBJECTS:TILEDB_VCF_OBJECTS>
 )
+if (WIN32)
+  # tiledb_vcf_unit ref's symbols that are not explicitly part of the exported public interface.
+  # Since Windows/MSVC does not make all symbols in a shared library visible,
+  # Use a cmake feature on a target specifically for tiledb_vcf_unit to export all symbols.
+  add_library(tiledbvcf4test SHARED
+    $<TARGET_OBJECTS:TILEDB_VCF_OBJECTS>
+  )
+  set_property(TARGET tiledbvcf4test PROPERTY WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif()
 
 target_link_libraries(tiledbvcf
   PUBLIC
     HTSlib::HTSlib
     TileDB::tiledb_shared
-    spdlog::spdlog
+# TBD: removing this avoids msvc/windows spdlog build issues, why is it here, why does removing this help?
```

Comment:
Does other platform CI pass without this?

Reply:
yes, other platforms are passing
it -seems- that on windows spdlog -is- being built as ...HEADER_ONLY even tho it should default to ...COMPILED_LIB, and I have thus far not been able to figure out why*, but with that ...HEADER_ONLY build state and the inclusion of spdlog in two locations, wind up with linker complaining about duplicate symbols.
([*] I've added debugging to ..vcf build and to examples build of spdlog itself, and the properties I -think- should be affecting seem to be the same in both cases, altho the spdlog examples build does -not- take the "bad" ...HEADER_ONLY conditional path.)
Note that this is -not- windows specific, at least on my wsl ubuntu 20.04 installation, that same ...HEADER_ONLY path is also being taken in the unix build, but I'm assuming ld is simply not complaining about the duplicate symbols.)